Indexing/SubstitutionTree.cpp

```diff
@@ -259,9 +259,10 @@ void SubstitutionTree::insert(Node** pnode,BindingMap& svBindings,LeafData ld)
     BindingMap::Iterator svit(svBindings);
     BinaryHeap<Binding, BindingComparator> remainingBindings;
     while (svit.hasNext()) {
```

Comment:
There's a default constructor for Binding (now killed) that leaves things unitialised. We can actually get rid of it altogether in this case.